contrib/initramfs/scripts/zfs.in

```diff
@@ -411,15 +411,21 @@ decrypt_fs()
 
 		# Determine dataset that holds key for root dataset
 		ENCRYPTIONROOT="$(get_fs_value "${fs}" encryptionroot)"
+		KEYLOCATION="$(${ZFS} get -H -o value keylocation "${ENCRYPTIONROOT}")"
```

Comment:
To stay uniform with the rest of the script, use get_fs_value(), like the line above it

Reply:
Good point, changed.